### ISSUE TITLE
i can use it

### ISSUE THREAD

**Issue by patchoka2011** [HenenTheProgrammer/Chat-App-in-PHP-MySQL#1]
Hello Bro 
i upload it on my pc but i cant use it whene open on browser
 thnx bro

**Comment by dev-henen** [comment]
Certainly! Here's a step-by-step guide on how to access the application:

**1. Install XAMPP:**
   Make sure XAMPP is installed on your PC. You can download it from the official website: [XAMPP](https://www.apachefriends.org/index.html).

2. **Check PHP Version:**
   Ensure that PHP version 5 and above is installed on your system. You can check your PHP version by running the following command in the command prompt:

   ```bash
   php -v
   ```

3. **Create a Database:**
   Open XAMPP and start the Apache and MySQL services. Then, create a new database in phpMyAdmin or any MySQL client tool.

4. **Import Database.sql:**
   Locate the `Database.sql` file in the `sql` folder of the application. Import this SQL file into the database you just created.

5. **Configure Database Connection:**
   Open the `config.php` file in the application's folder and set the database configuration parameters such as hostname, username, password, and database name.


6. **Set Environment Variable:**
   Add the PHP executable path to the system environment variables. For example, set the path to `C:\xampp\php`.

7. **Run the Application:**
   Open the command prompt and navigate to the folder where your application is located. Run the following command:

   ```bash
   php -S localhost:3000
   ```

   This command starts a built-in PHP web server on `localhost:3000`.

8. **Access the Application:**
   Open a web browser and navigate to `http://localhost:3000` to access your application.

Ensure that all the mentioned steps are followed carefully to successfully set up and run the application. If any issues arise, double-check the configurations and make necessary adjustments.

**Comment by patchoka2011** [comment]
Hello 
I'm using Parrot Os :'( i dnt understand if u can explaine More 
Thnx Bro

**Comment by dev-henen** [comment]
Here's an adjusted guide:

1. **Install XAMPP Equivalent:**
   Parrot OS may not use XAMPP. Instead, you can install the required components separately. Use the package manager (likely `apt`) to install Apache, MySQL, and PHP.

   ```bash
   sudo apt update
   sudo apt install apache2 mysql-server php
   ```

2. **Check PHP Version:**
   Confirm the installed PHP version with:

   ```bash
   php -v
   ```

3. **Create a Database:**
   Start the MySQL service and create a new database using the MySQL command-line or a tool like phpMyAdmin.

4. **Import Database.sql:**
   Import the SQL file into the database:

   ```bash
   mysql -u your_username -p your_database_name < path/to/Database.sql
   ```

5. **Configure Database Connection:**
   Open the `config.php` file and set the database configuration parameters.

6. **Run the Application:**
   Open the terminal, navigate to your application's folder, and start the PHP built-in server.

   ```bash
   php -S localhost:3000
   ```

7. **Access the Application:**
   Open a web browser and navigate to `http://localhost:3000` to access your application.

Make sure to adapt the commands and configurations based on the specific details of the Parrot OS environment. If any issues arise, refer to the Parrot OS documentation or community forums for additional guidance.

**Comment by patchoka2011** [comment]
Hello Bro 
i Upload it on a Server but i get blank page when i access to index.php

**Comment by dev-henen** [comment]
You are the one chatting with me, right?
